.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb"
+}
```

Comment:
EOF error here.

Reply:
@Betty-Kebenei Yah I know the format I was just suggesting a template to ease the PR process

Reply:
I have worked on that